config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_sqlinstances.sql.cnrm.cloud.google.com.yaml

```diff
@@ -527,6 +527,13 @@ spec:
                         type: array
                       requireSsl:
                         type: boolean
+                      sslMode:
+                        description: Specify how SSL connection should be enforced
+                          in DB connections. This field provides more SSL enforcment
+                          options compared to require_ssl. To change this field, also
```

Comment:
We probably want to change the field name to `requireSsl` in the description because in KCC resource, field names are in camel case. This can be achieved / "hacked" by updating the terraform lib.

Reply:
Good catch, updated the description

Yes, I created tests in `third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/services/sql/resource_sql_database_instance_test.go` and ran them successfully.